x-pack/packages/lock-manager/kibana.jsonc

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "shared-server",
+  "id": "@kbn/lock-manager",
+  "owner": ["@elastic/obs-ai-assistant"],  
+  "devOnly": true
```

Comment:
Is this a devOnly package or is it intended to be used in the final distributable?

Reply:
No, it's meant to be in the final distro. I'll remove this.